codeworld-api/src/CodeWorld/Driver.hs

```diff
@@ -728,11 +729,45 @@ picToObj' pic =
             return $ (unsafeCoerce arr :: JSVal)
     setProps xs obj = liftIO $ void $ mapM (\(s, v) -> setProp s v obj) xs
 
+-- describePicture 
+describePicture :: Picture -> String
+describePicture (Rectangle _ w h) = printf "rectangle { width = %4f , height = %4f }" w h
+describePicture (SolidPolygon _ pts) = printf "solidPolygon " ++ intercalate "," [printf "(%4f, %4f)" x y | (x,y) <- pts]
+describePicture (SolidClosedCurve _ pts) = printf "solidClosedCurve " ++ intercalate "," [printf "(%4f, %4f)" x y | (x,y) <- pts]
+describePicture (Polygon _ pts) = printf "polygon " ++ intercalate "," [printf "(%4f, %4f)" x y | (x,y) <- pts]
+describePicture (ThickPolygon _ pts w) = printf "thickPolygon " ++ intercalate "," [printf "(%4f, %4f)" x y | (x,y) <- pts] ++ printf ", {width = %4f}" w
+describePicture (SolidRectangle _ w h) = printf "solidRectangle { width = %4f , height = %4f }" w h
+describePicture (ThickRectangle _ lw w h) = printf "thickRectangle { linewidth = %4f , width = %4f , height = %4f }" lw w h
+describePicture (ClosedCurve _ pts) = printf "closedCurve" ++ intercalate "," [printf "(%4f, %4f)" x y | (x,y) <- pts]
+describePicture(ThickClosedCurve _ pts w) = printf "thickClosedCurve " ++ intercalate "," [printf "(%4f, %4f)" x y | (x,y) <- pts] ++ printf ", {width = %4f}" w
+describePicture (Circle _ r) = printf "circle { radius = %4f }" r
+describePicture (SolidCircle _ r) = printf "solidCircle { radius = %4f }" r
+describePicture (ThickCircle _ lw r) = printf "thickCircle { linewidth = %4f , radius = %4f }" lw r
+describePicture (Polyline _ pts) = printf "polyline " ++ intercalate "," [printf "(%4f, %4f)" x y | (x,y) <- pts]
+describePicture (ThickPolyline _ pts w) = printf "thickPolyline" ++ intercalate "," [printf "(%4f, %4f)" x y | (x,y) <- pts] ++ printf ", {width = %4f}" w
+describePicture (Curve _ pts) = printf "thickCurve" ++ intercalate "," [printf "(%4f, %4f)" x y | (x,y) <- pts]
+describePicture (ThickCurve _ pts w) = printf "thickCurve " ++ intercalate "," [printf "(%4f, %4f)" x y | (x,y) <- pts] ++ printf ", {width = %4f}" w
+describePicture (Sector _ b e r) = printf "sector { startAngle = %4f , endAngle = %4f , radius = %4f}" b e r
+describePicture (Arc _ b e r) = printf "arc { startAngle = %4f , endAngle = %4f , radius = %4f}" b e r
+describePicture (ThickArc _ b e r w) = printf "thickArc { startAngle = %4f , endAngle = %4f , radius = %4f , width = %4f}" b e r w
```

Comment:
Maybe call this `linewidth`, for consistency?

Reply:
for ```thickCurve``` or ```thickArc``` or both?